en/docs/learn/logging-in-to-opencart-using-the-identity-server.md

```diff
@@ -0,0 +1,131 @@
+    ![opencart-miniorange-add-layout](../assets/img/tutorials/opencart-miniorange-add-layout.png)
+    
+3.  Save the settings after adding the module to the display.
+
+## Configuring the service provider in WSO2 Identity Server
+
+1.  Sign in to the WSO2 Identity Server [Management Console](../../setup/getting-started-with-the-management-console/).
+
+2.  On the **Main** menu, click **Identity > Service Providers > Add**.
+
+3.  Fill in the **Service Provider Name** and provide a brief **Description** of the service provider. Only 
+Service Provider Name is a required field and you can use Opencart-SP as the name for this example.
+
+4.  Expand **Claim Configuration**.
+    1. Select **Use Local Claim Dialect**.
+    2. For **Requested Claims**, add the following claim URIs.
+        1. https://wso2.org/claims/lastname
+        2. https://wso2.org/claims/givenname
+    3. Set **Subject Claim URI** to https://wso2.org/claims/emailaddress.
+    
+    ![opencart-is-claim-config](../assets/img/tutorials/opencart-is-claim-config.png)
+    
+5.  Expand the **Inbound Authentication Configuration > SAML2 Web SSO Configuration** section and click **Configure**.
+    In the form that appears, fill out the following configuration details required for single sign-on. 
+    For more details on these attributes, refer 
+    [SAML2 Web SSO Configuration](../../learn/configuring-inbound-authentication-for-a-service-provider#configuring-inbound-authentication-with-saml2-web-sso).
+    1. For the value of **Issuer**, provide the SP Entity ID obtained as Service Provider Metadata when configuring SAML SSO extension in OpenCart.
+    2. For the value of **Assertion Consumer URL**, provide the ACS Url obtained as Service Provider Metadata when configuring SAML SSO extension in OpenCart. 
+    3. Uncheck **Enable Signature Validation in Authentication Requests and Logout Requests**.
+    4. Check **Enable Attribute Profile** and **Include Attributes in the Response Always**.
+    5. Save the configuration.
+
+    ![opencart-is-saml-sp-config](../assets/img/tutorials/opencart-is-saml-sp-config.png)
+
+## Try it out
+   
+1.  Visit the OpenCart site and click on login.
+    ![opencart-login](../assets/img/tutorials/opencart-login.png)
+
+2.  In the next view click on ‘Login with $app’ where $app is the application name you have provided when configuring 
+the SSO extension.
+    ![opencart-sso-login-button](../assets/img/tutorials/opencart-sso-login-button.png)
+        
+3.  You will be redirected to WSO2 Identity Server login page. Login by providing credentials of an user in WSO2 IS.
+    ![opencart-is-sso-login](../assets/img/tutorials/opencart-is-sso-login.png)
+	
+4.  Upon successful login you will be logged into OpenCart.
+    ![opencart-sso-success.png](../assets/img/tutorials/opencart-sso-success.png)
+    
+5.  Under your account personal details the user profile attributes configured in WSO2 Identity Server will be populated.
```

Comment:
Shall we change this to, 

The user profile attributes configured in WSO2 Identity Server will be populated in the Personal details of your account.